news/4.34/platform.html

```diff
@@ -181,6 +181,39 @@ <h2>General Updates </h2>
         </p>
       </td>
     </tr>
+    
+    <tr id="edge-browser-improvements"> <!-- https://github.com/eclipse-platform/eclipse.platform.swt/issues/1466 -->
+      <td class="title">Improvements to Edge Browser (Windows only)</td>
+      <td class="content">
+        <p>
+          In an effort to make it the default browser for Windows, several issues that affected the Edge browser 
+          have been addressed in this release. Some of them are:
+          <ul>
+          	<li>Added support for basic authentication</li>
+          	<li>Inherited the proper light/dark theme from Eclipse instead of Windows</li>
```

Comment:
Formulation is a bit misleading, as we did not inherit something but we _changed_ the inheritance behavior.
```suggestion
          	<li>Corrected theme inheritance to use proper light/dark theme from Eclipse instead of Windows</li>
```

Reply:
I like the current text better.

Reply:
The current formulation is still incorrect, as we did not inherit anything.